test/legacy/reconfigure.js

```diff
@@ -141,6 +141,31 @@ describe('Reconfigure', function() {
     done();
   });
 
+  it('gets URL by instance name alias from vcap containing single service', function(done) {
+    var config = { vcapInstanceName: 'serviceA', // alias of 'instanceName'
+      vcapServices: { cloudantNoSQLDB: [
+        { name: 'serviceA', credentials: { url: 'http://mykey:mypassword@mydomain.cloudant.com' } }
```

Comment:
Should we have the same `vcapServices` object design as the one in java-cloudant with the URI, username, and password?
https://github.com/cloudant/java-cloudant/blob/master/cloudant-client/src/test/java/com/cloudant/tests/CloudFoundryServiceTest.java#L66

Reply:
Yeah, it'd be nice but there are quite a few of these already. I reckon we should do this when we need to change the vcap stuff for IAM.

Reply:
Ok sounds good.